src/index.html

```diff
@@ -8,5 +8,40 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--first-month-day--tue calendar--last-month-day--30">
```

Comment:
```suggestion
      <div class="calendar calendar--tue calendar--30">
```
Не пиши название модификатора в классе, хватит просто `--tue` и `--30`

Reply:
![Screenshot_4](https://user-images.githubusercontent.com/73542090/121368854-0bed6b00-c944-11eb-9a65-ad729b6ce687.png)

Reply:
когда поменял модификатор, то не могу понять как теперь тут написать, теперь не работает как прежде

Reply:
.calendar--tue, .calendar--30, вместо той записи писал так и не получается